test/controllers/actions/FakeAuthAction.scala

```diff
@@ -57,6 +63,9 @@ object FakeAuthAction extends AuthAction {
 }
 
 class TestAuthAction extends AuthAction {
+  override val executionContext: ExecutionContext = ???
+  override val parser: BodyParser[AnyContent] = ???
```

Comment:
Why are these not implemented?

Reply:
It's just need to be overridden when you extends AuthAction. FakeAuthAction is for test purposes and both of those vals are not used. Implementation can be empty.